cbrain_task/bids_app_handler/bourreau/bids_app_handler.rb

```diff
@@ -0,0 +1,226 @@
+
+# A subclass of CbrainTask::ClusterTask to run BidsAppHandler.
+class CbrainTask::BidsAppHandler < ClusterTask
+
+  Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
+
+  include RestartableTask
+  include RecoverableTask
+
+  Bosh_Required_Version = "0.5.18" # 0.5.18 fails too; waiting for patches, 2019-02-13
```

Comment:
So is this checking against your current special version of Bosh?

Reply:
Nope :-( It's just asking for 0.5.18, which I expected I would be able to work around, but even a plain 0.5.18 won't work.